Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
+  z-index: 2147483647;
+  position: absolute;
 }
 
-span.DropDownMenuSelect.disabled
+ul.DropDownMenuOptions
 {
-  cursor: default;
-  border: 1px solid #BCBCBC;
+  box-shadow: var(--remotion-themed-box-shadow);
+  padding: var(--remotion-themed-padding);
+  list-style: none;
+  background-color: var(--color-disabled-background);
 }
 
-span.DropDownMenuSelect span
+span.DropDownMenuItemIcon
 {
-  left: 0;
-  vertical-align: middle;
+  width: 1.25em;
 }
 
-span.DropDownMenuSelect a.DropDownMenuLabel
+/* style separator */
+span.DropDownMenuSeparator
 {
-  position: relative;
-  left: 0.1em;
-  top: 0.1em;
-  margin-right: 25px;
-  margin-right: calc(22px + 0.1em);
-  margin-left: 4px;
-  border-style: none;
-  background-color: Transparent;
-  color: Black;
-  font-weight: bold;
-  height: 1.2em !important;
+  display: flex;
+  line-height: var(--remotion-themed-separator-height);
+  margin: var(--remotion-themed-margin) 0;
+  background-color: var(--color-contrast-separator);
 }
 
-span.DropDownMenuSelect a.DropDownMenuLabel:focus
+div.DropDownMenuStatus
 {
-  outline: none;
-  text-decoration: underline;
+  overflow: hidden;
+  /* int max for highest z-index possible */
+  z-index: 2147483647;
+  /* position and dimension are set via javascript */
+  max-height: 4em;
+  max-width: 30em;
 }
 
-span.DropDownMenuContainer span.disabled a.DropDownMenuLabel
+/* TODO: rework indicator when new design available */
+div.DropDownMenuStatus > div
 {
-  color: Gray;
+  width: 100%;
+  padding: 0.25em;
+  border: 1px solid Black;
+  background-color: Lightyellow;
+  box-sizing: border-box;
 }
 
-span.DropDownMenuSelect a.DropDownMenuButton
+div.DropDownMenuStatus.loading > div
 {
-  display: inline-block;
-  position: absolute;
-  left: auto;
-  top: 0px;
-  bottom: 0;
-  right: 0px;
-  border-left: solid 1px #AAAAAA;
-  margin-right: 0;
+  background-image: url('../Image/sprite.svg#DropDownMenuRequestInProgressIndicator');
+  background-size: 16px 16px;
+  background-repeat: no-repeat;
+  background-position: 0.25em;
+  padding-left: calc(16px + 0.25em);
+  min-height: 20px;
 }
 
-span.DropDownMenuSelect.focus a.DropDownMenuButton,
-span.DropDownMenuSelect:hover a.DropDownMenuButton
+div.DropDownMenuStatus.error > div
 {
-  border-left: solid 1px #7CB2E6;
+  background-image: url('../Image/sprite.svg#DropDownMenuErrorIndicator');
+  background-size: 16px 16px;
+  background-repeat: no-repeat;
+  background-position: 0.25em;
+  padding-left: calc(16px + 0.5em);
+  min-height: 20px;
 }
 
-span.DropDownMenuSelect.nestedHover:hover a.DropDownMenuButton
+ul.DropDownMenuOptions li a
 {
-  border-left: solid 1px #AAAAAA;
+  cursor: pointer;
+  text-decoration: none;
+  display: flex;
+  padding: var(--remotion-themed-padding);
+  white-space: nowrap;
+  color: Black;
+  outline: none;
 }
 
-span.DropDownMenuSelect:active a.DropDownMenuButton
+ul.DropDownMenuOptions li a span
 {
-  border-left: 1px solid #569CE0;
+  display: inline-flex;
```

Comment:
I haven't played around with it, but there seens to be a lot of nested flex-boxes. Do we actually need those? 

Can we just make a flexgrid or css grid inside the anchor, the img-span and the text-span are both grid-items and the items are gapped and padded?

Reply:
I removed all but one inline-flex and I changed the anchor to grid and its children to grid-items.